docs/reference/search.rst

```diff
@@ -85,3 +85,47 @@ The current implementation can be expensive if you have a lot of entities as the
 .. note::
 
     There is a work in progress to use an async JavaScript solution to better load data from the database.
+
+Customize visibility of empty result boxes
+------------------------------------------
+
+By default all the admin boxes are show in search results and it looks like:
+
+.. figure:: ../images/empty_boxes_show.png
+    :align: center
+    :alt: Custom view
+    :width: 700px
+
+We can fade out the boxes that have no results with:
+
+.. code-block:: yaml
+
+    # app/config/config.yml
```

Comment:
shouldn't this be something like sonata_config, or it is okay like that?

Reply:
Well, if `config/packages/sonata_admin.yaml` is more correct, then yes IMO.

Reply:
should I edit the whole document then and fix it everywhere? `config/packages/sonata_admin.yaml` is correct if you are using flex.